types/dompurify/index.d.ts

```diff
@@ -10,15 +10,25 @@
 // Minimum TypeScript Version: 4.5
 /// <reference types="trusted-types"/>
 
-import { DOMWindow } from 'jsdom';
```

Comment:
please also remove dependency of jsdom! thanks!

Reply:
If you mean manually removing the dependency in `package.json` or something, it is my understanding that the deleted line you commented on will already remove this dependency as soon as this PR is merged (see my original PR in #64622 - I did not need to edit `package.json` myself, DefinitelyTyped automation handles that)